src/views/standalone/network/InterfacesAndDevicesView.vue

```diff
@@ -1,5 +1,5 @@
+Copyright (C) 2024 Nethesis S.r.l.
 <!--
-  Copyright (C) 2024 Nethesis S.r.l.
   SPDX-License-Identifier: GPL-3.0-or-later
 -->
```

Comment:
?

Reply:
# :man_facepalming: 

I'm surprised it still compiles